formatTest/unit_tests/expected_output/modules.re

```diff
@@ -445,9 +447,8 @@ let module Id (X: Type) => X;
 let module Compose
            (F: Type => Type)
            (G: Type => Type)
-           (X: Type) => F (
-  G X
-);
+           (X: Type) =>
+  F (G X);
```

Comment:
I was suggesting to treat functor args the same as function args, but as far as I understand this code isn't formatted the same as it would be as a function instead of a functor.  In particular, wouldn't function args be indented only 2 spaces, rather than being aligned under the name?